test/format/ops.c

```diff
@@ -1164,11 +1164,23 @@ ops(void *arg)
                     if (tinfo->last > max_rows)
                         tinfo->last = 0;
                     /*
-                     * Edge case: If we are truncating to the end of the table and we're doing
-                     * mirroring with both a VLCS and FLCS table, then we can get a false mismatch
-                     * failure if the truncate happens before the first append-only insert to the
-                     * VLCS table. So if we have VLCS and FLCS mirrors, set last to the second last
-                     * key.
+                     * Edge case: There is a case where we cannot detect a proper mirror mismatch.
+                     * Say we truncated the tail end key range of all the mirrors from N-max_rows.
```

Comment:
I'd change `N-max_rows` to something like `N to max_rows` to avoid possible confusion the hyphen is meant as a minus sign.